lampod-cli/src/args.rs

```diff
@@ -79,13 +79,13 @@ impl TryInto<LampoConf> for LampoCliArgs {
             conf.node = node.clone();
         }
         if self.bitcoind_url.is_some() {
-            conf.core_url = self.bitcoind_url;
+            conf.core_url = self.bitcoind_url.unwrap();
         }
```

Comment:
`.unwrap()` cause program panic! again and as in all my review you should avoid to use it:)

it is so clean write it in the following way

```rust
if let Some(bitcoin_url) = self.bitcoin_url {
 conf.core_url = bitcoin_url;
}
```